mkt/constants/applications.py

```diff
@@ -29,7 +29,15 @@ class DEVICE_GAIA(object):
     api_name = 'firefoxos'
 
 
-DEVICE_TYPE_LIST = [DEVICE_DESKTOP, DEVICE_MOBILE, DEVICE_TABLET, DEVICE_GAIA]
+class DEVICE_TV(object):
+    id = 5
+    name = _(u'Firefox OS TV')
+    class_name = 'firefoxos+tv'
+    api_name = 'firefoxos+tv'
```

Comment:
can't we just call it ```tv``` such that we don't have to differentiate it with FirefoxOS